custom/mlservermllib/models/utils.py

```diff
@@ -0,0 +1,35 @@
+from mlserver import ModelSettings
+from .errors import InvalidMLlibFormat
+from pyspark.mllib.classification import LogisticRegressionModel, NaiveBayesModel, SVMModel
+from pyspark.mllib.clustering import KMeansModel
+from pyspark.mllib.regression import IsotonicRegressionModel, LinearRegressionModel, LassoModel, RidgeRegressionModel
+from pyspark.mllib.tree import DecisionTreeModel, GradientBoostedTreesModel, RandomForestModel
+
+
+async def get_mllib_load(settings: ModelSettings):
+    if not settings.parameters:
+        raise InvalidMLlibFormat(settings.name)
+
+    mllib_format = settings.parameters.format
+
+    if not mllib_format:
+        raise InvalidMLlibFormat(settings.name)
+
+    model_dict = {
```

Comment:
Would it make sense to move this dict into a "const" (as in a "fake const", like `MODEL_DICT`)?